jobs/pull-kubernetes-federation-e2e-gce-canary.env

```diff
@@ -0,0 +1,27 @@
+# Panic if anything mutates a shared informer cache
+ENABLE_CACHE_MUTATION_DETECTOR=true
+
+FEDERATION=true
+USE_KUBEFED=true
+PROJECT=k8s-jkns-pr-bldr-e2e-gce-fdrtn
```

Comment:
You need a different project. I don't want to risk running things in the same project.